api/rest/v1/bulk_variables.md

```diff
@@ -0,0 +1,363 @@
+---
+layout: default
+title: Variables
+parent: v1 REST
+grand_parent: API
+nav_exclude: true
+published: false
+permalink: /api/rest/v1/bulk/variables
+---
+
+# /v1/bulk/variables
+
+Get all [variables](/glossary.html#variable) associated with a specific entity, for multiple entities.
```

Comment:
"associated" can be expanded a little bit, like "variables with data associated ..."?

Reply:
Done!